packages/Config/RectorConfig.php

```diff
@@ -373,8 +373,10 @@ private function isRuleNoLongerExists(mixed $skipRule): bool
             is_string($skipRule)
             // not regex path
             && ! str_contains($skipRule, '*')
-            // not realpath
-            && realpath($skipRule) === false
+            // not directory
+            && ! is_dir($skipRule)
+            // not file
+            && ! is_file($skipRule)
             // a Rector end
             && str_ends_with($skipRule, 'Rector')
```

Comment:
We should do this cheap chrck earlier

Reply:
https://github.com/rectorphp/rector-src/pull/5039/commits/8fae3786555cc27787bbfd9a1dfe6f41068376f6